Assets/Materials/CraterTown/transparentGlass.mat

```diff
@@ -24,10 +24,7 @@ Material:
   m_Shader: {fileID: -6465566751694194690, guid: 6dfb8dada865e684ab7b6b4c4131b184, type: 3}
   m_Parent: {fileID: 0}
   m_ModifiedSerializedProperties: 0
-  m_ValidKeywords:
-  - MAIN_LIGHT_CALCULATE_SHADOWS
-  - _MAIN_LIGHT_SHADOWS_CASCADE
-  - _SHADOWS_SOFT
+  m_ValidKeywords: []
```

Comment:
oh whoops, should have made custom material for the vials on the brick barrel :skull: 
this is used for the glass in grand canyon so please create a new one here? ...or, I could do it myself I suppose :/